browser/extensions/api/brave_rewards_api.cc

```diff
@@ -644,6 +645,12 @@ void BraveRewardsGetRewardsParametersFunction::OnGetRewardsParameters(
   }
   data.Set("walletProviderRegions", std::move(provider_regions));
 
+  if (!parameters->vbat_deadline.is_null()) {
+    data.Set("vbatDeadline", floor(parameters->vbat_deadline.ToDoubleT() *
```

Comment:
nit: it is recommended when `base::Value` to use the helpers in `//base/json/values_util.h` for serialization

Reply:
See TimeToValue and ValueToTime